mesonbuild/ast/interpreter.py

```diff
@@ -273,15 +273,12 @@ def reduce_arguments(
                 key_resolver: T.Callable[[mparser.BaseNode], str] = default_resolve_key,
                 duplicate_key_error: T.Optional[str] = None,
             ) -> T.Tuple[T.List[TYPE_nvar], TYPE_nkwargs]:
-        if isinstance(args, ArgumentNode):
```

Comment:
You're going to have to work quite hard to convince me that removing this isinstance check is "removing a typing comment". ;)